src/Components/MonthPicker.js

```diff
@@ -0,0 +1,55 @@
+import DropDownPicker from 'react-native-dropdown-picker';
+import React, { useState } from 'react';
+import { monthName } from '../functions/monthName';
+
+export const MonthPicker = ({setTime}) => {
+    const currentMonth = new Date().getMonth() + 1; 
+    var month3;
+    const currentYear = new Date().getFullYear();;
+    switch(currentMonth) {
+        case 1:
+            month3 = 10;
+            break;
+        case 2:
+            month3 = 11;
+            break;
+        case 3:
+            month3 = 12;
+            break;
+        default:
+            month3 = currentMonth - 3;
+    }
+
+    const [open, setOpen] = useState(false);
+    const [items, setItems] = useState([
+        {label: 'Month', value: 'Month'},
+        {label: `${monthName(month3)}`, value: `${month3}`, parent: 'Month'},
+        {label: `${monthName(month3 + 1 % 12)}`, value: `${month3 + 1 % 12}`, parent: 'Month'},
+        {label: `${monthName(month3 + 2 % 12)}`, value: `${month3 + 2 % 12}`, parent: 'Month'},
```

Comment:
I might be wrong but `month3 + 1 % 12` seems to be the same as `month3 + 1`

Reply:
It's to wrap around the months! For example if the month3 is December, then I want the next month to be January and not an unknown month 13